mkdocs.yml

```diff
@@ -49,6 +49,7 @@ extra_css:
     
 plugins:
   - search
+  - with-pdf
```

Comment:
This addition is causing the [check docs](https://github.com/planetlabs/planet-client-python/actions/runs/8354173225/job/22867090232?pr=1027#step:5:51) build step to fail, because `with-pdf` is not installed.  I haven't estimated the level of effort for adding this plugin; do we need it here or is it safe to remove `with-pdf` from this PR and include it in a follow up PR?

Reply:
Ah, my bad. That was an experiment I did (and forgot about - I haven't used my local repo in a year). I'll remove.

Reply:
Removed. The plugin was to be able to output a big pdf to then be able to put all the docs into a custom GPT, which was nice. I think it did depend on some alternate installation (like not easily done with pip install), so probably not worth the effort of including in the main stuff. It was more of an experiment, I'd just totally forgot about it - even when I saw the build error.